src/transaction/log_recovery_analysis.cpp

```diff
@@ -1719,6 +1751,9 @@ log_rv_analysis_record_on_tran_server (THREAD_ENTRY *thread_p, LOG_RECTYPE log_t
       (void) log_rv_analysis_atomic_sysop_start (thread_p, tran_id, log_lsa);
       break;
 
+    case LOG_ASSIGNED_MVCCID:
+      break;
```

Comment:
call `log_rv_analysis_assigned_mvccid` or am I completely missing the point...

Reply:
Good catch :). Fixed.